config.sample.yaml

```diff
@@ -53,6 +53,12 @@ tre:
   # Set to true if TreAdmins should be able to assign and de-assign users to workspaces via the UI
   user_management_enabled: false
 
+  # Uncomment to enable DNS Security policy on the system, and add any known DNS names that you need to allow DNS queries on.
+  # Note, these are need to be fully qualified, i.e. they end in a dot(.)
+  # enable_dns_policy: true
+  # allowed_dns:
+  #   - mydomain.com.
+
```

Comment:
Can you add details to the docs please. 

Also these are additional to the system defaults?